cookiecutter.json

```diff
@@ -1,15 +1,18 @@
 {
-  "app_name": "template",
-  "app_title": "Application template",
-  "short_description": "The default output of the AiiDAlab application cutter.",
-  "github_user": "aiidalab",
-  "repo_name": "aiidalab-{{cookiecutter.app_name}}",
+  "app_name": "my-app",
+  "app_title": "My App",
+  "short_description": "This AiiDAlab application was created with the app cutter.",
+  "github_username": "aiidalab",
+  "repo_name": "aiidalab-{{ cookiecutter.app_name.lower().replace(' ', '-').replace('_', '-')}}",
```

Comment:
is the app name allowed to contain spaces?
if not I would rather prefer to warn if it does (the cookiecutter allows for such checks I believe)

Reply:
I'll have a look.

Reply:
Thanks!